### ISSUE TITLE
reference dataset, same commit id for all SCC and SCS

### ISSUE THREAD

**Issue by jgse** [eiffel-community/eiffel#296]
### Description
When exploring the reference dataset, found all the SCC and SCS use the same commit id.
As I understand, new SCC/SCS should have a new commit id, or anything I misunderstood?

**Comment by magnusbaeck** [comment]
Yes, normally there would be one SCC (and maybe an SCS) for every (commit, branch, repo) combination. If multiple branches contain the same commit there could certainly be multiple SCCs with the same commit id. I suspect the reference data set's use of the same commit id everywhere is an oversight.

**Comment by jgse** [comment]
Yes,  should be 1-1 mapping between SCC(+SCS) and the `(commit, branch, repo)` combination.

but in reference data set, many SCC/SCS have same` (commit, branch, repo) `combination.
for example, with below condition, I expect only one SCC or SCS return, but it returns many
```python
        "gitIdentifier": {
          "commitId": "fd090b60a4aedc5161da9c035a49b14a319829b4",
          "repoName": "myRepo",
          "branch": "master",
          "repoUri": "https://repo.com"
       }
```
and the differences in those events are the value of "iteration" in "customData". 
```python
"customData": [
      
          {
            "value": 1,
            "key": "iteration"
          }
        ],
```

**Comment by m-linner-ericsson** [comment]
I would also agree it is a bug in https://github.com/eiffel-community/eiffel/blob/master/examples/reference-data-sets/default/events.zip
